includes/admin/forms/class-metabox-form-data.php

```diff
@@ -100,6 +100,26 @@ function get_settings() {
 		$prefix = '_give_';
 
 		$settings = array(
+			/**
+			 * Theme Options
+			 */
+			'form_template_options' => apply_filters(
+				'give_form_template_options',
+				array(
+					'id'        => 'form_template_options',
+					'title'     => __( 'Form Template', 'give' ),
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.

Reply:
@bradp This whole file need to update. I will create  separate issue for this.